src/components/frame/Extensions/pages/ExtensionDetailsPage/partials/ExtensionDetails/hooks.tsx

```diff
@@ -1,12 +1,14 @@
 import React, { useCallback, useMemo, useState } from 'react';
 import { format } from 'date-fns';
+import { toast } from 'react-toastify';
 import { ExtensionStatusBadgeMode } from '~common/Extensions/ExtensionStatusBadge/types';
```

Comment:
Add blank space between imports packages and other tools like types, components, hooks etc.